src/main/java/uk/gov/hmcts/ccd/v2/V2.java

```diff
@@ -8,7 +8,8 @@
     public final class MediaType {
         // External API
         public static final String CASE = "application/vnd.uk.gov.hmcts.ccd-data-store-api.case.v2+json;charset=UTF-8";
-        public static final String START_TRIGGER = "application/vnd.uk.gov.hmcts.ccd-data-store-api.start-trigger.v2+json;charset=UTF-8";
+        public static final String START_CASE_TRIGGER = "application/vnd.uk.gov.hmcts.ccd-data-store-api.start-case-trigger.v2+json;charset=UTF-8";
```

Comment:
We are ok to change this as nobody is consuming the start-trigger.v2 endpoint yet.